erpnext/templates/pages/order.js

```diff
@@ -39,4 +40,24 @@ frappe.ready(function(){
 			});
 		}
 	}
-})
+
+});
+
+function initiate_payment() {
+	let gateway_url = doc_info.payment_gateway_url;
+
+	if (frappe.cart.selected_payment_gateway) {
+		fetch("/api/resource/Payment%20Gateway%20Account?filters=" + `[["payment_gateway","=","${frappe.cart.selected_payment_gateway}"]]`)
```

Comment:
Maybe we should pre-cache this value, not sure. What do you think, @forellana-digithinkit?

Reply:
Yeah agreed, this call and redirect only adds delay to the ux. Specially on a slow server/connection